Quoting ChangeLog of X11's videoproto package:
vldXvMC.h: stop uselessly including XvMClib.h

--- src/video_out/xxmc.h
+++ src/video_out/xxmc.h
@@ -77,10 +77,10 @@
 #include <X11/extensions/XShm.h>
 #include <X11/extensions/Xv.h>
 #include <X11/extensions/Xvlib.h>
+#include <X11/extensions/XvMClib.h>
 #ifdef HAVE_VLDXVMC
   #include <X11/extensions/vldXvMC.h>
 #else
-  #include <X11/extensions/XvMClib.h>
   #include <X11/extensions/XvMC.h>
 #endif
 
